### PR TITLE
Documentation tweaks

### LINKED ISSUE #117: Issues when creating new project
Here's a list of bits and pieces I encountered when creating a new project from scratch:

- In "create-new" the current command `copier copy gh:DiamondLightSource/python-copier-template /path/to/my-project` needs `--trust` added to it, the same as is present on the main README
- It's confusing if you try and create a project with a hypen or dash in the name - the various fields will auto-correct themselves to make the names compliant with the relevant specs for each question, but it feels like there should be something to suggest what _should_ be done in this case
- The "Upload to GitHub" section doesn't work as is, as there are no commits in the repository. The instructions should either explicitly say to do an initial commit, or the copier template should have done it automatically
- When trying to do a commit I see the error `pre-commit configuration detected, but `pre-commit install` was never run`.
- When trying to install pre-commit, "ruff" is not found. I think the summary is that you need to install the entire project (including dev dependencies) before doing an initial commit?
    - I had an additional issue where the pre-commit was causing commits to become impossible to do. The symptom was that during commits I saw the below pre-commit message, and all commits would fail. The fix was to run `pre-commit install -f` to do a forced install and override some legacy config somewhere.
```
git commit -m "initial commit"
pre-commit configuration detected, but `pre-commit install` was never run
check for added large files..............................................Passed
check yaml...............................................................Passed
check for merge conflicts................................................Passed
lint with ruff...........................................................Passed
format with ruff.........................................................Passed
(venv) [eyh46967@pc0103 psc-datalogger]$ git log
fatal: your current branch 'main' does not have any commits yet
```

- The Project Description is not edited through the Settings menu, it's done from the cogwheel on the main page next to the "About" header

### LINKED ISSUE #128: Documentation needs to specify Python version required
On Diamond machines if you `pip install copier` in a venv created with the `python3` command it will give you a very outdated version of the module, using `module load python` first should be recommended.

### PR DESCRIPTION
**Pull request by coretl**
Fixes #128, partially fixes #117

### CODE REVIEW

File: copier.yml [hunk 1]
```diff
@@ -40,7 +44,8 @@ distribution_name:
     type: str
     help: |
         Name of the python distribution package that will be created.
-        This is what people will `pip install`
+        This is what people will `pip install`.
+        It is generally the same as the repo name unless there is a name clash on PyPI.
```
Comment (AlexanderWells-diamond) [inline copier.yml:48]: VERY predantic but it's odd to have the "repo_name" help start with "Generally" and this one start with "It is generally" as it mixes styles.

File: docs/tutorials/create-new.md [hunk 2]
```diff
@@ -4,24 +4,35 @@ Once you have followed the [](./installation) tutorial, you can use `copier` to
...
 
 You can now [create a new blank project on GitHub](https://github.com/new). Choose the same GitHub owner and repo name that you answered in the questions earlier. GitHub will now give you the commands needed to upload your repo from GitHub
 
-## Setings
+## Settings
+
+You can now go to the cogwheel on the main page next to the "About" header, and set the project description to match the answer you gave.
 
-You can now go to the `Settings` and set:
+Then go to the `Settings` and set:
```
Comment (AlexanderWells-diamond) [inline docs/tutorials/create-new.md:34]:
```suggestion
Then go to the `Settings` tab and set:
```

**Review by AlexanderWells-diamond — APPROVED** [review]
Apart from a couple of very pedantic comments this all looks good to me.

File: copier.yml (hunk 1 above)
Comment (coretl) [reply copier.yml:48]:
```suggestion
        Generally the same as the repo name unless there is a name clash on PyPI.
```
How about this?

Comment (AlexanderWells-diamond) [reply copier.yml:48]: Good with me

**Review by AlexanderWells-diamond — APPROVED** [review]

**Pull request merged**